.env.sample

```diff
@@ -116,5 +116,6 @@ CLAMAV_SCANNING_ENABLED=false
 FF_TFM_PAYMENT_RECONCILIATION_ENABLED=false
 FF_TFM_FACILITY_END_DATE_ENABLED=false
 FF_TFM_DEAL_CANCELLATION_ENABLED=false
+AUTOMATIC_SF_CUSTOMER_CREATION_ENABLED=false
```

Comment:
I think we should have `SALESFORCE` instead of `SF`, initially, I didn't know what SF meant